java/test/src/main/java/test/Ice/stream/Test.ice

```diff
@@ -22,7 +22,7 @@ interface MyInterface;
 
 ["java:serializable:test.Ice.stream.Serialize.Small"] sequence<byte> SerialSmall;
 
-struct SmallStruct
+struct LargeStruct
```

Comment:
It's not clear to me why you renamed this struct. In C++, we have the same test and the struct is still named SmallStruct.

Reply:
I renamed it to be more accurate. This struct is actually quite large with how many fields it has in it!
Part of why I didn't find this until now is because I saw "SmallStruct" and thought "oh there's no way this involves classes".

For reference, the other tests in which we use `SmallStruct` look like:
```
struct SmallStruct
{
    byte m;
}
```

Normally I wouldn't really care, but since the name did have an impact on my searching, I figured it best to make it more accurate. Should I revert it, or fix the other stream tests? Either option would take a couple minutes...

Reply:
I'll remove it from this PR and do it in a separate PR.
I agree, it's outside the scope of this PR.